### PR TITLE
Update documentation and examples

### PR DESCRIPTION
**Pull request by kkujawa-sumo**

### CODE REVIEW

File: .github/workflows/pull_requests.yml [hunk 1]
```diff
@@ -89,7 +89,7 @@ jobs:
         run: make deploy-examples
       - name: Deploy example with configuration in CRD
```
Comment (perk-sumo) [inline .github/workflows/pull_requests.yml:90]: Shouldn't this be a `Deploy example with configuration in CR` then?

File: operator/docs/configuration.md [hunk 2]
```diff
@@ -27,6 +29,53 @@ metadata:
...
+used by the operator.
+
+Example definitions of `TailingSidecarConfig` are available in [samples](../config/samples) directory and Pod adjusted to this configuration
+in available [pod-with-tailing-sidecar-config.yaml](../examples/pod_with_tailing_sidecar_config.yaml).
+
+To try example configuration use following commands:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/SumoLogic/tailing-sidecar/main/operator/config/samples/tailing-sidecar_v1_tailingsidecar.yaml
+kubectl apply -f https://raw.githubusercontent.com/SumoLogic/tailing-sidecar/main/operator/examples/pod_with_śtailing_sidecar_config.yaml
```
Comment (perk-sumo) [inline operator/docs/configuration.md:49]:
```suggestion
kubectl apply -f https://raw.githubusercontent.com/SumoLogic/tailing-sidecar/main/operator/examples/pod_with_tailing_sidecar_config.yaml
```

File: .github/workflows/pull_requests.yml (hunk 1 above)
Comment (perk-sumo) [reply .github/workflows/pull_requests.yml:90]:
```suggestion
      - name: Deploy example with configuration in CR
```

File: .github/workflows/pull_requests.yml [hunk 3]
```diff
@@ -100,7 +100,7 @@ jobs:
         run: make deploy-examples-update
       - name: Deploy update for example with configuration in CRD
```
Comment (perk-sumo) [inline .github/workflows/pull_requests.yml:101]:
```suggestion
      - name: Deploy update for example with configuration in CR
```

File: .github/workflows/pull_requests.yml [hunk 4]
```diff
@@ -147,7 +147,7 @@ jobs:
         run: make deploy-examples
       - name: Deploy example with configuration in CRD
```
Comment (perk-sumo) [inline .github/workflows/pull_requests.yml:148]:
```suggestion
      - name: Deploy example with configuration in CR
```

File: .github/workflows/pull_requests.yml [hunk 5]
```diff
@@ -160,7 +160,7 @@ jobs:
         run: make deploy-examples-update
       - name: Deploy update for example with configuration in CRD
```
Comment (perk-sumo) [inline .github/workflows/pull_requests.yml:161]:
```suggestion
      - name: Deploy update for example with configuration in CR
```

File: .github/workflows/pull_requests.yml [hunk 6]
```diff
@@ -210,7 +210,7 @@ jobs:
         run: make deploy-examples
       - name: Deploy example with configuration in CRD
```
Comment (perk-sumo) [inline .github/workflows/pull_requests.yml:211]:
```suggestion
      - name: Deploy example with configuration in CR
```

File: .github/workflows/pull_requests.yml [hunk 7]
```diff
@@ -223,7 +223,7 @@ jobs:
         run: make deploy-examples-update
       - name: Deploy update for example with configuration in CRD
```
Comment (perk-sumo) [inline .github/workflows/pull_requests.yml:224]:
```suggestion
      - name: Deploy update for example with configuration in CR
```

File: operator/docs/configuration.md (hunk 2 above)
Comment (kkujawa-sumo) [reply operator/docs/configuration.md]: Changed in https://github.com/SumoLogic/tailing-sidecar/pull/162/commits/32aaf3e15ae19592a75529f0e4f4742c3477da20

File: .github/workflows/pull_requests.yml (hunk 6 above)
Comment (kkujawa-sumo) [reply .github/workflows/pull_requests.yml]: Changed in https://github.com/SumoLogic/tailing-sidecar/pull/162/commits/38f885df52f94adcc40a1f3a3a0a1daad46697e9

File: .github/workflows/pull_requests.yml (hunk 7 above)
Comment (kkujawa-sumo) [reply .github/workflows/pull_requests.yml]: Changed in https://github.com/SumoLogic/tailing-sidecar/pull/162/commits/38f885df52f94adcc40a1f3a3a0a1daad46697e9

File: .github/workflows/pull_requests.yml (hunk 5 above)
Comment (kkujawa-sumo) [reply .github/workflows/pull_requests.yml]: Changed in https://github.com/SumoLogic/tailing-sidecar/pull/162/commits/38f885df52f94adcc40a1f3a3a0a1daad46697e9

File: .github/workflows/pull_requests.yml (hunk 3 above)
Comment (kkujawa-sumo) [reply .github/workflows/pull_requests.yml]: Changed in https://github.com/SumoLogic/tailing-sidecar/pull/162/commits/38f885df52f94adcc40a1f3a3a0a1daad46697e9

File: .github/workflows/pull_requests.yml (hunk 4 above)
Comment (kkujawa-sumo) [reply .github/workflows/pull_requests.yml]: Changed in https://github.com/SumoLogic/tailing-sidecar/pull/162/commits/38f885df52f94adcc40a1f3a3a0a1daad46697e9

File: .github/workflows/pull_requests.yml (hunk 1 above)
Comment (kkujawa-sumo) [reply .github/workflows/pull_requests.yml]: Changed in https://github.com/SumoLogic/tailing-sidecar/pull/162/commits/38f885df52f94adcc40a1f3a3a0a1daad46697e9

**Review by perk-sumo — APPROVED** [review]
🚀

**Pull request merged**